docs/usage/user-stories/swissquote.md

```diff
@@ -174,7 +174,7 @@ Some features and options we enjoy:
 
 - [Shared configurations (presets)](../key-concepts/presets.md), we have a Swissquote default configuration we set for all repositories, each team can extend it with their own practices
 - [Integration with GitHub’s Dependabot alerts](../configuration-options.md#vulnerabilityalerts) to raise the priority and send security remediation PRs as soon as possible
-- Each rule can be customized either globally [or specified per package](../configuration-options/#packagerules)
+- Each rule can be customized either globally [or specified per package](../configuration-options.md#packagerules)
```

Comment:
The bad link is published in the production site, when I click it there I end up at a 404 page.

Our docs-build step passes, but I was expecting it to fail on this bad link.

https://github.com/renovatebot/renovatebot.github.io/blob/98e8361aefdb1423e617f8d016ae3a08cee0d1ab/mkdocs.yml#L3-L5